source/operations_and_monitoring.rst

```diff
@@ -113,8 +113,8 @@ On each controller:
 Some services may store data in a dedicated Docker volume, which can be removed
 with ``docker volume rm``.
 
-Installing and Updating TLS Certificates
-----------------------------------------
+Installing TLS Certificates
+---------------------------
 
 |tls_setup|
 
```

Comment:
Related: I find it easier to copy the cert to etc/kayobe/kolla/certificates/haproxy.pem